pkg/client/services/redhatopenshift/mgmt/2020-04-30/redhatopenshift/client.go

```diff
@@ -26,7 +26,8 @@ import (
 
 const (
 	// DefaultBaseURI is the default URI used for the service Redhatopenshift
-	DefaultBaseURI = "https://management.azure.com"
+	// DefaultBaseURI = "https://management.azure.com"
+	DefaultBaseURI = "https://localhost:8443"
```

Comment:
This should be overridden in the tests, not in SDK. It is global var so should be possible to override from test context.